awslabs/aws-lambda-powertools-typescript
# fix(commons): rename tests subfolder to samples to avoid being deleted by tools such as node-prune

## Description of your changes

Current project structure is having a folder called `tests` under `commons/src` package and was hosting samples context and events to use when  calling a lambda function.

Customers using tools like [node-prune](https://github.com/tj/node-prune) where having issue (#881) because the tool was deleting the folder. A fix would have been to simply use the `-exclude` option of this tool but here we propose to simply rename the folder to `samples`.


### How to verify this change

1. prep the env
   ```sh
   npm ci --foreground-scripts
   npm run lerna-package
   cd examples/cdk
   npm ci
   npm install ../../packages/**/dist/aws-lambda-powertools-*
   npm run test
   ```
1. use node-prune
   ```
   node-prune -exclude '*.d.ts' -excludes 'assets'
   ```
1. test again, they should still pass
   ```sh
   npm run test
   ```

### Related issues, RFCs

#881 

### PR status

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist


---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
